floating-buttons.html

```diff
@@ -9,14 +9,16 @@
 				box-sizing: border-box;
 				display: block;
 			}
-			:host > .container {
+			.container {
```

Comment:
Do these class names and their rules end up in the global namespace? i.e. should we be more specific with their names?